### ISSUE TITLE
Adjust integration tests to test out-of-order File resource creation or file upload

### ISSUE THREAD

**Issue by ofanobilbao** [HumanCellAtlas/ingest-central#115]
i.e Test that we can successfully submit an envelope if the Files are uploaded before any metadata is created, and visa versa

Acceptance criteria:
Adjust the integration test(s) such that the following sequences of actions are carried out in order:
- a submission is created
- a file is uploaded
- the spreadsheet is uploaded
- the rest of the files are uploaded

**Comment by justincc** [comment]
Possibly duplicated by #212

**Comment by justincc** [comment]
@aaclan-ebi Is this still relevant?

**Comment by aaclan-ebi** [comment]
yes, we have a unit test for this, but it's better if we can add a test case for the integration test.
